.eslintrc

```diff
@@ -1,11 +1,30 @@
 {
 	"extends": [
-		"xo",
-		"xo-typescript",
-		"xo-react",
+		"eslint:recommended",
+		"plugin:react/recommended",
+		"plugin:@typescript-eslint/recommended",
+		"plugin:prettier/recommended"
 	],
+	"plugins": [
+		"react",
+		"@typescript-eslint",
+		"prettier"
+	],
+	"env": {
+		"browser": true,
+		"jasmine": true,
+		"jest": true
```

Comment:
jasmine _and_ jest?

Reply:
transfer mistake^^ think also we can remove jasmine